src/LDLFactorizations.jl

```diff
@@ -128,13 +128,9 @@ function ldl_symbolic!(n, Ap, Ai, Lp, parent, Lnz, flag, P, Pinv)
 end
 
 function ldl_numeric_upper!(n, Ap, Ai, Ax, Cp, Ci, Lp, parent, Lnz, Li, Lx, D, Y,
-                            pattern, flag, P, Pinv; dynamic_args...)
-  if length(dynamic_args) != 0
-    dynamic_regul = true
-    dynamic_dict = Dict(dynamic_args)
-  else
-    dynamic_regul = false
-  end
+                            pattern, flag, P, Pinv, r1, r2, tol, n_d)
```

Comment:
I was thinking keyword arguments, i.e.,
```julia
pattern, flag, P, Pinv; r1=zero(eltype(Ax)), r2=zero(eltype(Ax)), tol=zero(eltype(Ax)), n_d=n)
```
Positional arguments work and you can give them default values too. The disadvantage is that if you only want to change `n_d`, you must also specify all the others.

Reply:
Nevermind, you're using keyword arguments below already.

Reply:
oups sorry I did not refresh the page, should I undo my commit?